plugins/services/src/js/components/HighOrderServiceInstances.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import Pod from "../structs/Pod";
+import PodInstancesContainer
+  from "../containers/pod-instances/PodInstancesContainer";
+import Service from "../structs/Service";
+import ServiceInstancesContainer
+  from "../containers/service-instances/ServiceInstancesContainer";
+
+const HighOrderServiceInstances = props => {
+  const { pod, service, params } = props;
+  if (pod != null) {
+    return <PodInstancesContainer pod={pod} />;
+  }
+
+  return <ServiceInstancesContainer service={service} params={params} />;
+};
+
+HighOrderServiceInstances.propTypes = {
```

Comment:
Please reduce the number of properties. As a user of the component I shouldn't need to worry wether my service is a pod or an app.